src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +185,114 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[TagsDict], session: Session) -> None:
```

Comment:
is there not a method in `sagemaker.session.Session` that we can use to make the low-level `sagemaker.add_tags` call?